cypress-tests/cypress/e2e/PaymentUtils/WellsFargo.js

```diff
@@ -0,0 +1,492 @@
+                status: 200,
+                body: {
+                    status: "succeeded",
+                },
+            },
+        },
+        Capture: {
+            Request: {
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "USD",
+                customer_acceptance: null,
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "succeeded",
+                    amount: 6500,
+                    amount_capturable: 0,
+                    amount_received: 6500,
+                },
+            },
+        },
+        PartialCapture: {
+            Request: {},
+            Response: {
+                status: 200,
+                body: {
+                    status: "partially_captured",
+                    amount: 6500,
+                    amount_capturable: 0,
+                    amount_received: 100,
+                },
+            },
+        },
+        Void: {
+            Request: {},
+            Response: {
+                status: 200,
+                body: {
+                    status: "cancelled",
+                },
+            },
+        },
+        Refund: {
+            Request: {
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "USD",
+                customer_acceptance: null,
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "pending",
+                },
+            },
+        },
+        PartialRefund: {
+            Request: {
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "USD",
+                customer_acceptance: null,
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "pending",
+                },
+            },
+        },
+        SyncRefund: {
+            Request: {
+                payment_method_data: {
+                    card: successfulNo3DSCardDetails,
+                },
+                currency: "USD",
+                customer_acceptance: null,
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "pending",
+                },
+            },
+        },
+        MandateSingleUse3DSAutoCapture: {
+            Request: {
+                payment_method_data: {
+                    card: successfulThreeDSTestCardDetails,
+                },
+                currency: "USD",
+                mandate_data: singleUseMandateData,
+            },
+            Response: {
+                status: 200,
+                body: {
+                    status: "succeeded",
+                },
+            },
+        },
```

Comment:
Does `wellsFargo` support mandates? If not, you can remove.

Reply:
Yes it does